bigtable/gc.go

```diff
@@ -25,12 +25,27 @@ import (
 	bttdpb "google.golang.org/genproto/googleapis/bigtable/admin/v2"
 )
 
+type PolicyType int
+
+const (
+	PolicyTypeUnknown PolicyType = iota
```

Comment:
When would the PolicyType be unknown?

Reply:
It'll be for `NoGcPolicy`. I think the naming could be better, like PolicyTypeUnspecified to match with no policy, but follow the suggestion by tritone@.